terraform/aws/environments/dev/self_build_params.auto.tfvars

```diff
@@ -0,0 +1,12 @@
+# Generated from release version - using prebuilt AMI
```

Comment:
This file was generated by https://github.com/privacysandbox/aggregation-service/blob/main/terraform/aws/fetch_terraform.sh